.github/workflows/build-push.yml

```
@@ -0,0 +1,43 @@
+        description: 'Build without cache'
+        type: boolean
+        default: false
+
+jobs:
+  build:
+    runs-on: 
+      - windows-latest
+      # - ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)